addons/website_forum/static/src/js/tours/website_forum.js

```diff
@@ -63,5 +63,72 @@ odoo.define("website_forum.tour_forum", function (require) {
         trigger: ".o_wforum_validate_toggler[data-karma=\"20\"]:first",
         content: _t("Click here to accept this answer."),
         position: "right",
+    }
+    ]);
+
+    tour.register("flag_question", {
```

Comment:
This should be called in python to be part of the testing suite..
This will:
1. Strengthen our testing suite
2. Make sure this tour is not broken one day without noticing it